File: mycode.py

def append(list, item):
    list.append(item)
def listget(list, index):
    return list[index]
    
def checkNumber(number):
    if number%3==0:
        return "fizz"
    elif number%5==0:
        return "buzz"
    elif number%5==0 and number%3==0:
        return "fizzbuzz"
    return str(number)
def doFizzbuzz(number):
    _ = print(number)
    _ = print(checkNumber)
    _ = print("")
def main():
    _ = doFizzbuzz(5)
    _ = doFizzbuzz(3)
    _ = doFizzbuzz(15)
    _ = doFizzbuzz(2)
main()

File: mycode.c

void esll_print(int text) {
    printf(text);
}

char* esll_checkNumber(int esll_number)
{
    if (esll_number%3==0)
    {
        return "fizz";
    }
    else if (esll_number%5==0)
    {
        return "buzz";
    }
    else if (esll_number%5==0 && esll_number%3==0)
    {
        return "fizzbuzz";
}
    return esll_number;
}
void esll_doFizzbuzz(int esll_number)
{
    esll_print(esll_number);
    esll_print(esll_checkNumber);
    esll_print("");
}
void esll_main()
{
    int esll_makeVariableTest = 3;
    esll_doFizzbuzz(5);
    esll_doFizzbuzz(3);
    esll_doFizzbuzz(15);
    esll_doFizzbuzz(2);
}

int main() {
    esll_main();
}

